DDDEastAnglia.Tests/SponsorBuilder.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using DDDEastAnglia.DataAccess.SimpleData.Models;
+
+namespace DDDEastAnglia.Tests
+{
+    internal class SponsorBuilder
+    {
+        private byte[] logo;
+        private string name;
+        private DateTime? paymentDate;
+        private bool showPublicly = true;
+        private int sponsorId;
+        private int sponsorshipAmount;
+        private string url;
+
+        public Sponsor Build()
```

Comment:
That's true, but in reality over the past two years we haven't been able to group the sponsors that neatly into those amounts. Even though we have the levels, many sponsors don't fit it that closely.